staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -21,14 +21,15 @@ import (
 	"strconv"
 	"time"
 
-	"k8s.io/apimachinery/pkg/util/clock"
 	"k8s.io/apiserver/pkg/server/mux"
 	"k8s.io/apiserver/pkg/util/flowcontrol/counter"
 	fq "k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing"
+	fairqueuingclock "k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/clock"
 	fqs "k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset"
 	"k8s.io/apiserver/pkg/util/flowcontrol/metrics"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/klog/v2"
+	utilclock "k8s.io/utils/clock"
```

Comment:
I wouldn't bother to rename this import, let it be known as "clock".  We are trying to move everything that was using `k8s.io/apimachinery/pkg/util/clock` to `k8s.io/utils/clock` so that we can retire `k8s.io/apimachinery/pkg/util/clock`.

Reply:
we have two clock packages being imported here - `fairqueuing` and `utils`, so the prefix makes it more explicit, but i am fine either way.

Reply:
`k8s.io/utils/clock` is the base clock package.  `k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/clock` builds on that, adding `EventClock`.  Now that I think of it, it would be better if the latter were `k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/eventclock`.

Reply:
@MikeSpreitzer  yes, i agree, `eventclock` is a better name - can we merge this PR? I am happy to open a separate PR with the rename.

Reply:
Mike already have one opened.